_posts/en/newsletters/2023-06-14-newsletter.md

```diff
@@ -0,0 +1,200 @@
+---
+title: 'Bitcoin Optech Newsletter #255'
+permalink: /en/newsletters/2023/06/14/
+name: 2023-06-14-newsletter
+slug: 2023-06-14-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter summarizes a discussion about allowing relay of
+transactions containing data in the taproot annex field and links to a
+draft BIP for silent payments.  Also included is another entry in our
+limited weekly series about mempool policy, plus our regular sections
+summarizing a Bitcoin Core PR Review Club meeting, announcing new
+software releases and release candidates, and describing notable changes
+to popular Bitcoin infrastructure software.
+
+## News
+
+- **Discussion about the taproot annex:** Joost Jager [posted][jager
+  annex] to the Bitcoin-Dev a request for a change in the Bitcoin Core
+  transaction relay and mining policy to allow storing arbitrary data in
+  the [taproot][topic taproot] annex field.  This field is an optional
+  part of the witness data for taproot transactions.  If present,
+  signatures in taproot and [tapscript][topic tapscript] must commit to
+  its data (making it impossible for a third party to add, remove, or
+  change) but it has no other defined purpose at present---it's reserved
+  for future protocol upgrades, especially soft forks.
+
+    Although there have been [previous proposals][riard annex] to define
+    a format for the annex, these have not seen widespread acceptance
+    and implementation.  Jager proposed two formats ([1][jager annex],
+    [2][jager annex2]) that could be used to allow anyone to add
+    arbitrary data to the annex in a way that would not significantly
+    complicate later standardization efforts that might be bundled with
+    a soft fork.
+
+    Greg Sanders [replied][sanders annex] to ask what data Jager
+    specifically wanted to store in the annex and described his own use
+    for the annex in testing the [LN-Symmetry][topic eltoo] protocol
+    with the [SIGHASH_ANYPREVOUT][topic sighash_anyprevout] soft fork
+    proposal using Bitcoin Inquisition (see [Newsletter #244][news244
+    annex]).  Sanders also described a problem with the annex: in a
+    multi-party protocol (such as a [coinjoin][topic coinjoin]), each
+    signature only commits to the annex for the input containing that
+    signature---not the annexes for other inputs in the same
+    transaction.  That means if Alice, Bob, and Mallory sign a coinjoin
+    together, there's no way Alice and Bob can prevent Mallory from
+    broadcasting a version of the transaction with a large annex that
+    delays its confirmation.  Because Bitcoin Core and other full nodes
+    don't currently relay transactions that contain annexes, this is not
+    a problem at present.  Jager [replied][jager annex4] that he wants
+    to store signatures from ephemeral keys for a type of [vault][topic
+    vaults] that doesn't require a soft fork, and he  [suggested][jager
+    annex3] that some
```

Comment:
One addition here could be that the annex seems to be the only way to create the vault and store the ephemeral signer signatures in one atomic operation in bitcoin currently.

Reply:
@joostjager thanks for the feedback, but since we are so close to publishing, Im going to leave this out for now.